providers/openebs/bulk-upgrade/cstor_replica_version.yml

```diff
@@ -5,8 +5,6 @@
                 -o=jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}'
               register: cvr_name
               failed_when: 'cvr_name.stdout == ""'
-              with_items:
```

Comment:
why it is removed

Reply:
As we were parsing this value as outer loop we no need this items here